src/Dfe.PlanTech.Infrastructure.Data/CmsDbContext.cs

```diff
@@ -167,14 +168,18 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
 
         modelBuilder.Entity<ButtonWithEntryReferenceDbEntity>(entity =>
         {
+            entity.ToView("ButtonWithEntryReferencesWithSlug");
             entity.Navigation(button => button.Button).AutoInclude();
+            entity.Property(button => button.LinkType)
+                .HasConversion(linkType => linkType.ToString(), linkType => (LinkToEntryType)Enum.Parse(typeof(LinkToEntryType), linkType))
+                .Metadata.SetAfterSaveBehavior(PropertySaveBehavior.Ignore);
+
+            entity.Property(button => button.Slug).Metadata.SetAfterSaveBehavior(PropertySaveBehavior.Ignore);
```

Comment:
I don't quite understand what this is doing, why do you need to set this behaviour?

Reply:
It makes EF Core not try to save the properties to the DB if they're changed.

So if the `ButtonWithEntryReference` receives an update via the webhook, without these it'll try to save these columns (which will be null - because they don't actually exist on the content type), and will cause a whoopsee.